src/client/pages/Landing/landingPageData.ts

```diff
@@ -0,0 +1,87 @@
+import { Market } from 'components/utils/CurrentLocale'
+
+type Locale = 'se' | 'se-en' | 'no' | 'no-en' | 'dk' | 'dk-en'
+
+type AlternateLinksData = {
+  hrefLang: string
+  locale: Locale
+}[]
+
+export const alternateLinksData: AlternateLinksData = [
+  { hrefLang: 'sv-se', locale: 'se' },
+  { hrefLang: 'en-se', locale: 'se-en' },
+  { hrefLang: 'nb-no', locale: 'no' },
+  { hrefLang: 'en-no', locale: 'no-en' },
+  { hrefLang: 'da-dk', locale: 'dk' },
+  { hrefLang: 'en-dk', locale: 'dk-en' },
+]
```

Comment:
This feels like something we can use in other places so maybe move it to a generic locales file or WDYT? No need to change this now however :)

Reply:
Yes my idea is that we should have this kind of structure more generally and yes this specific array is probably useful as it is in other places! But also I think we could do that separately, it might get messy in this branch otherwise 😄